packages/app/src/app/pages/NewDashboard/Content/index.tsx

```diff
@@ -8,27 +8,33 @@
 // import SearchSandboxes from './routes/SearchSandboxes';
 // import CreateTeam from './routes/CreateTeam';
 // import TeamView from './routes/TeamView';
+import { Element } from '@codesandbox/components';
+import { StartSandboxes } from './routes/StartSandboxes';
+import { Templates } from './routes/Templates';
 
-// const Content = () => (
-//   <Switch>
-//     <Route path="/dashboard/recent" component={RecentSandboxes} />
-//     <Route path="/dashboard/trash" component={DeletedSandboxes} />
-//     <Route path="/dashboard/templates" exact component={Templates} />
-//     <Route path="/dashboard/sandboxes/:path*" component={PathedSandboxes} />
-//     <Route path="/dashboard/search" component={SearchSandboxes} />
-//     <Route path="/dashboard/teams/new" component={CreateTeam} />
-//     <Route exact path="/dashboard/teams/:teamId" component={TeamView} />
-//     <Route
-//       path="/dashboard/teams/:teamId/sandboxes/:path*"
-//       component={PathedSandboxes}
-//     />
-//     <Route
-//       path="/dashboard/teams/:teamId/templates"
-//       component={Templates}
-//       exact
-//     />
-//     <Redirect to="/dashboard/recent" />
-//   </Switch>
-// );
+const ContentComponent = () => (
+  <Element style={{ width: 960, margin: '40px auto' }}>
+    <Switch>
+      <Route path="/new-dashboard/start" component={StartSandboxes} />
+      <Route path="/new-dashboard/templates" component={Templates} />
+      {/* <Route path="/dashboard/trash" component={DeletedSandboxes} />
+    <Route path="/dashboard/templates" exact component={Templates} />
+    <Route path="/dashboard/sandboxes/:path*" component={PathedSandboxes} />
+    <Route path="/dashboard/search" component={SearchSandboxes} />
+    <Route path="/dashboard/teams/new" component={CreateTeam} />
+    <Route exact path="/dashboard/teams/:teamId" component={TeamView} />
+    <Route
+      path="/dashboard/teams/:teamId/sandboxes/:path*"
+      component={PathedSandboxes}
+    />
+    <Route
+      path="/dashboard/teams/:teamId/templates"
+      component={Templates}
+      exact
+    /> */}
+      <Redirect to="/new-dashboard/start" />
+    </Switch>
+  </Element>
+);
 
-// export default withRouter(Content);
+export const Content = withRouter(ContentComponent);
```

Comment:
No need for `withRouter` here, since we're not using any of the passed props.
https://reacttraining.com/react-router/core/api/withRouter

```suggestion
```